src/Modules/SimplCommerce.Module.Localization/Controllers/LocalizationController.cs

```diff
@@ -10,11 +13,12 @@ public class LocalizationController : Controller
         [HttpPost]
         public IActionResult SetLanguage(string culture, string returnUrl)
         {
-            Response.Cookies.Append(
-                CookieRequestCultureProvider.DefaultCookieName,
-                CookieRequestCultureProvider.MakeCookieValue(new RequestCulture(culture)),
-                new CookieOptions { Expires = DateTime.UtcNow.AddYears(1) }
-            );
+            var cultureRepository = HttpContext.RequestServices.GetRequiredService<IRepositoryWithTypedId<Culture, string>>();
```

Comment:
cultureRepository should be injected in the constructor

Reply:
Yep